services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -103,10 +102,10 @@ createBrigUser suid (Id buid) teamid mbName managedBy = do
     Just n -> pure n
     Nothing -> do
       let subject = suid ^. SAML.uidSubject
-          badName = throwSpar . SparBadUserName $ SAML.encodeElem subject
-          mkName  = Name . fromRange <$>
-                    (fmap (Text.take 128) . SAML.shortShowNameID >=> checked @ST @1 @128) subject
-      maybe badName pure mkName
+          uname   = Text.take 128 . SAML.unsafeShowNameID $ subject
```

Comment:
1. Why do we silently truncate the name instead of failing?

2. When an unsafe function is used, there should be a comment explaining why it's fine in this case.

Reply:
Please extract the `128` into a top-level value so you can give it a nice name like `maxUsernameLength`; this'll encourage consistency and 'self-documents' what the number is; right now it's a [Magic Number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)